src/main/java/ch/uzh/ifi/hase/soprafs24/controller/Controller.java

```diff
@@ -47,7 +47,7 @@ public String joinSession(@RequestBody JoinDTO joinData) {
         sessionService.validateSessionId(joinData.getSessionId());
 
         if (sessionService.hasStarted(joinData.getSessionId())) {
-            return null;
+            throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "The game has already started, cannot join anymore");
```

Comment:
I think this also need adaption on the client side.

Reply:
I'll check that bybthe end of today.

The client should just display error message and stay in title screen right?